lib/reek/smells/attribute.rb

```diff
@@ -9,22 +9,18 @@ module Smells
     # invites client classes to become too intimate with its inner workings,
     # and in particular with its representation of state.
     #
-    # Currently this detector raises a warning for every +attr+,
-    # +attr_reader+, +attr_writer+ and +attr_accessor+ -- including those
-    # that are private.
+    # This detector raises a warning for every public
+    # +attr_writer+, +attr_accessor+, and +attr+ with the writable set to true.
```

Comment:
Since this is RDoc format, the code-marking character is the plus. I think we use backticks in several places but that doesn't actually yield the proper result in the docs.